apps/dashboard/app/views/module_browser/_module_table.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="table-responsive">
+  <table class="table">
+    <thead class="table-light">
+      <tr>
+        <th></th>
+        <th>Name</th>
+        <th>Version(s)</th>
+        <th>Description</th>
```

Comment:
I don't think description should be a column. Very likely to overflow and just be noisy.

I'd say let the description appear when you open the module for further inspection.